src/java/org/apache/cassandra/db/SnapshotDetailsTabularData.java

```diff
@@ -69,14 +75,28 @@
     }
 
 
-    public static void from(final String snapshot, final String ks, final String cf, Map.Entry<String, Directories.SnapshotSizeDetails> snapshotDetail, TabularDataSupport result)
+    public static void from(final String snapshot, final File manifestFile, final String ks, final String cf, Map.Entry<String, Directories.SnapshotSizeDetails> snapshotDetail, TabularDataSupport result)
```

Comment:
@fibersel I think this might ideally also accept just expiresAt and createdAt parameters in order to avoid the parsing of file with all its complexities and it would be just simple stuff.